src/components/organisms/ClustersPane/ClustersPane.tsx

```diff
@@ -53,10 +64,12 @@ const ClustersPane = () => {
   const kubeconfig = useAppSelector(state => state.config.kubeconfigPath);
   const kubeConfig = useAppSelector(state => state.config.kubeConfig);
   const uiState = useAppSelector(state => state.ui);
-  const [currentKubeConfig, setCurrentKubeConfig] = useState<string>('');
 
+  const [currentKubeConfig, setCurrentKubeConfig] = useState<string>('');
   const fileInput = useRef<HTMLInputElement>(null);
 
+  const isEditingDisabled = uiState.isClusterDiffVisible;
```

Comment:
I think this should also be true if isInClusterMode

Reply:
done in new commit